tests/shared/src/test/scala/coursier/test/CentralTests.scala

```diff
@@ -790,6 +790,13 @@ abstract class CentralTests extends TestSuite {
       )
     }
 
+    'parentVersionInPom - {
+      * - resolutionCheck(
```

Comment:
(nitpicking) the `* - ` isn't necessary here, like in `groupIdVersionProperties` above. It just adds an extra level in the utest output (I see that `parentBeforeImports` does that below… but it shouldn't…).